shopinvader_sale_packaging_wishlist/tests/test_wishlist_packaging.py

```diff
@@ -2,12 +2,13 @@
 # @author Simone Orsi <simahawk@gmail.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-
 from odoo.addons.shopinvader_wishlist.tests.test_wishlist import (
     CommonWishlistCase,
 )
+from odoo.tests import tagged
 
 
+@tagged("post_install", "-at_install")
```

Comment:
@simahawk why to you run test at post_install? I'm not a fan since it's break the isolation between addons when running tests

Reply:
@simahawk Can we avoid running tests @post_install?